br/tests/lightning_max_incr/data/db.test1.000000000.csv

```diff
@@ -0,0 +1,3 @@
+"a","b"
+1,2
+9223372036854775807,3
```

Comment:
I think `2^53` should be a good value here to show the max auto value is far more smaller than `2^63-1` as expected.

Reply:
Why is 2^53 better? Though 2^63-1 is larger than the max_auto_random_base, it's still a valid value. Perhaps I could add a check to see if we could go on inserting tuples to test1 to prove auto_random_base is set to correct value so that the table remains available and fulfills constraints.

Reply:
Just a reminder, we should compare the `max_auto_random_base`, but not the final value. e.g. The `max_auto_random_base` of `(2^54)+1` is `1`.

Reply:
As `2^63-1` it the max value of int64, so it's obvious that any further allocate will cause overflow. But the available allocation range for autorandom is much smaller, so inserting with this value cause panic can't fully prove that the logic is correct. E.g, a implement the same as auto increment will also pass this check. BTW, you should use a different test case to show the difference between auto_increment and auto_random.

Reply:
Fix in 17b9c33